modular_skyrat/master_files/code/modules/mob/living/examine_tgui.dm

```diff
@@ -75,18 +75,18 @@
 	// Now we handle silicon and/or human, order doesn't really matter
 	// If other variants of mob/living need to be handled at some point, put them here
 	if(issilicon(holder))
```

Comment:
```suggestion
	if(preferences && issilicon(holder))
```

Reply:
do this instead of all the preferences? checks below

Reply:
But also if preferences is null you might as well refuse to open the UI or something, this whole proc pretty much relies on preferences being not null

Reply:
thanks boss